src/Gateways/AlipayHK.php

```diff
@@ -75,11 +75,24 @@ public function find($order): Collection
             $this->payload['timestamp']
         );
 
-        $this->payload['sign'] = Support::generateSign(array_except($this->payload, ['sign_type', 'sign']), $this->config->get('md5_key'));
+        $key = "";
+
+        $this->config->get('rsa_key') != null ? $key = $this->config->get('rsa_key') : $key = $this->config->get('md5_key');
```

Comment:
its extremely weird ternary operator usage. one of the most inappropriate usage as well.

```php
$key = "";
$this->config->get('rsa_key') != null ? $key = $this->config->get('rsa_key') : $key = $this->config->get('md5_key');

// vs 

$key = $this->config->get('rsa_key') ?: $this->config->get('md5_key');
```

see which is better n human-readable n clean?